src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardDefendantClaimMatcher.java

```diff
@@ -457,4 +407,24 @@ public boolean isCaseStayed() {
         return caseData.getCcdState() == CaseState.CASE_STAYED;
     }
 
+    @Override
+    public boolean isHearingScheduled() {
```

Comment:
This method and the next one are the same as the Claimant class (possibly more). Could they not go to the super class ?

Reply:
Should be resolved now